modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -66,15 +68,19 @@ NOTE: In Kubernetes, run the `transfer_leadership` request on the Pod that is ru
+
+For predictable and stable performance, Redpanda ensures an even distribution of a topic's partitions across all brokers in a cluster. It allocates partitions to random healthy brokers to prevent topic hotspots, without waiting for a batch of moves to finish before scheduling the next batch.
+
+Different topics in a cluster can have vastly different load profiles, but partitions of a single topic are often similar. To distribute the workload of high-demand topics, Redpanda first aims for an equal count of a topic's partitions on each broker. After that balances, Redpanda then aims for an equal overall partition count on each broker. This topic-aware partition balancing is enabled by default with the xref:reference:cluster-properties.adoc#partition_autobalancing_topic_aware[`partition_autobalancing_topic_aware`] property.
 
-For predictable and stable performance, Redpanda ensures that a topic's partitions are evenly distributed across all brokers in a cluster. It allocates partitions to random healthy brokers, to avoid topic hotspots, without needing to wait for a batch of moves to finish before it schedules the next batch.
+For example, suppose you have a Redpanda cluster that has 3 brokers and 1 topic `light` with 100 partitions and replication factor=3. Partitions are balanced, so each broker hosts 100 replicas of partitions of topic `light`. You expand the cluster with 3 new brokers and, before partition balancing moves partitions to the new broker, you create another topic `heavy`, which serves 10x more produce/consume requests than `light`. The `heavy` topic also has 100 partitions and replication factor=3. With topic-aware partition balancing, when creating `heavy`, Redpanda prioritizes an equal count of partitions: brokers 1, 2, and 3 host 100 replicas of `light` and 50 replicas of `heavy`, and brokers 4, 5, and 6 host 50 replicas of `heavy`. Partition balancing then kicks in and evens out the distribution of replicas of `light`, so that there are 50 replicas of `light` and 50 replicas of `heavy` on each broker.
 
 Redpanda supports flexible use of network bandwidth for replicating under-replicated partitions. For example, if only one partition is moving, it can use the entire bandwidth for the broker. Redpanda detects which shards are idle, so other shards can essentially steal bandwidth from them. Total bandwidth is controlled by the xref:reference:cluster-properties.adoc#raft_learner_recovery_rate[`raft_learner_recovery_rate`] property.
 
 Redpanda's default partition balancing includes the following:
 
-* When a broker is added to the cluster, some replicas are moved from other brokers to the new broker to take advantage of the additional capacity.
+* When a broker is added to the cluster, some replicas are moved from other brokers to the new broker to take advantage of the additional capacity. 
```

Comment:
Wondering why you need this need space at the end here?

Reply:
not needed: not sure why it was added!